apps/web/src/pages/SearchRequests/SearchPage/components/EstimatedCandidates.tsx

```diff
@@ -111,24 +111,26 @@ const EstimatedCandidates = ({
             ) : (
               <CandidateMessage candidateCount={candidateCount} />
             )}
-            <p>
-              <ScrollToLink to="results" color="black" mode="inline">
-                {candidateCount
-                  ? intl.formatMessage({
-                      defaultMessage: "View results",
-                      id: "3wbcnZ",
-                      description:
-                        "A link to view the pools that contain matching talent.",
-                    })
-                  : intl.formatMessage({
-                      defaultMessage:
-                        "Submit an empty request and we will try to help.",
-                      id: "9qzCX/",
-                      description:
-                        "Link text to scroll to the submit button when no candidates were found",
-                    })}
-              </ScrollToLink>
-            </p>
+            {!updatePending && (
```

Comment:
We're already checking updatePending right above this. Why not just move this in there?

Reply:
Oh I think I see what you mean, yeah I think that looks nicer
https://github.com/GCTC-NTGC/gc-digital-talent/pull/11703/commits/4c38ee41ca8048c076f2ba5a345dd53bd3dbcfd1